app/models/video.py

```diff
@@ -0,0 +1,22 @@
+from flask import current_app
+from app import db
+from sqlalchemy.orm import relationship
+
+
+class Video(db.Model):
+    __tablename__ = "video"
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime, nullable=True)
+    total_inventory = db.Column(db.Integer)
+    available_inventory = db.Column(db.Integer)
```

Comment:
Could you pull available_inventory from the rental table rather than storing it directly in this table?